sherlock-platform/customization/src/com/google/sherlock/onboarding/ProjectSettingsStep.java

```diff
@@ -0,0 +1,71 @@
+/***
+ * Copyright 2024 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***/
+
+package com.google.sherlock.onboarding;
+
+import com.intellij.ide.util.projectWizard.AbstractNewProjectStep;
+import com.intellij.ide.util.projectWizard.ProjectSettingsStepBase;
+import com.intellij.openapi.fileChooser.FileChooserDescriptorFactory;
+import com.intellij.openapi.ui.TextFieldWithBrowseButton;
+import com.intellij.platform.DirectoryProjectGenerator;
+import org.jetbrains.annotations.NotNull;
+
+import javax.swing.*;
```

Comment:
No wildcard imports. I suggest changing your IntelliJ IDEA Java/Kotlin language settings to disable this.